src/swapnaWJun20/RetrieveUniqueNameArray.java

```diff
@@ -0,0 +1,36 @@
+/*Assignment 35:  Retrieve all unique names from given array.
+
+String[] name1 = {"Palak", "Viresh", "Yash", "Aavruti"};
+String[] name2 = {"Deavina","Palak","Viresh", "Nikhil"};
+output : [Palak, Viresh, Yash, Aavruti,Deavina,Nikhil]*/
+
+package swapnaWJun20;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class RetrieveUniqueNameArray {
+
+	ArrayList<String> getUniqueElement(ArrayList<String> alName1, ArrayList<String> alName2) {
+		ArrayList<String> tempList = new ArrayList<String>(alName1);
+		ArrayList<String> outputList = new ArrayList<String>();
+		outputList.addAll(alName2);
```

Comment:
Here If we have a duplicate value in 2nd list then it will display both the duplicate list. Other wise code looks good.

Reply:
Thanks for the comment Anup, I have added the logic and updated the code